src/contracts/dev/LedaNFT-address.json

```diff
@@ -0,0 +1,3 @@
+{
+  "address": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
```

Comment:
Be careful with this

Reply:
Actually, I’m building this feature to not be careful with this